apps/expo/src/components/inputs/TextInput.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { TextInput as UnstyledTextInput, View } from "react-native";
+import type { TextInputProps, ViewProps } from "react-native";
+
+type CustomTextInputProps = TextInputProps;
+
+export const TextInput: React.FC<CustomTextInputProps> & {
+  RightElement: typeof RightElement;
+} = ({ children, ...props }) => {
+  return (
+    <View className="position-relative">
```

Comment:
should the TextInput be wrapped in a view? might be better to remove and let the parent control the styling?

Reply:
ok i removed the view. i updated the example usage in this PR to include the view when using the TextInput component because otherwise the icon won't be positioned properly